### PR TITLE
Chris santiago

### PR DESCRIPTION
**Pull request by resoluteCoder**
Completed the todo list app and news app. Ready for review

@marceloarb

### CODE REVIEW

File: todo/src/App.js [hunk 1]
```diff
@@ -1,29 +1,123 @@
...
+        InputGroup, Input, InputGroupAddon, 
+        Button, Container
+    } from 'reactstrap';
+import Tasks from './Components/Tasks'
+import Categories from './Components/Categories'
+import './styles.css';
 
 function App() {
+    const sampleData = [
+        {
```
Comment (marceloarb) [inline todo/src/App.js:18]: Where are you using this data? Also, if data is needed should be stored elsewhere.

File: todo/src/App.js [hunk 2]
```diff
@@ -1,29 +1,123 @@
...
+
+    const filterChecklist = (list) => list
+        .filter(task => task.category === activeCategory);
+
+    const removeAllTaskFromCategory = () => {
+        const remainingTasks = checklist
+            .filter(task => task.category !== activeCategory);
+        updateChecklist([...remainingTasks]);
+    }
+
```
Comment (marceloarb) [inline todo/src/App.js:48]: Too many functions inside your app function. This breaks the single responsibility principle. If a function like 'deleteCategory' or 'createCategory' is only used with the component category that function should be there.

File: todo/src/App.js (hunk 1 above)
Comment (resoluteCoder) [reply todo/src/App.js:18]: The data is for showing example tasks upon loading the app. It isn't required for the app to work.

File: todo/src/App.js (hunk 2 above)
Comment (resoluteCoder) [reply todo/src/App.js:48]: That makes sense. It seems I'm treating the 'App' function like a class in terms of syntax. Similar to how it was used before functional components could use 'useState'

**Comment by marceloarb** [comment]
Good job.

**Pull request merged**